src/components/StarRating.ts

```diff
@@ -36,15 +38,29 @@ export class StarRating extends Component<StarRatingProps, {}> {
         this.fractions = readOnly ? 2 : 1;
         this.stop = this.props.maximumStars;
 
-        return createElement("div", {
-            className: classNames("widget-star-rating", this.props.className),
-            style: this.props.style
-        },
+        const customStyle: object | undefined = this.props.starSize === "custom"
+            ? { fontSize: `${this.props.starSizeCustom}px` }
+            : undefined;
+        const EmptyStar = createElement("div", {
+            className: `glyphicon glyphicon-star-empty widget-star-rating-empty ` +
+                `widget-star-rating-font-${this.props.starSize}`,
+            style: customStyle
+        });
+        const FullStar = createElement("span", {
```

Comment:
Why is the full star a `span`, while the empty star is a `div`?

Reply:
Was done